charts/pravega-operator/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+name: pravega-operator
+version: 0.1.0
+appVersion: 2.6.0
```

Comment:
The `appVersion` is supposed to indicate the version of the Pravega Operator that this chart deploys.

Reply:
I think `appVersion` indicates the version of Pravega, not the version of the Pravega Operator.

Reply:
`appVersion` can't specify what version of Pravega it is because that's something the user specifies when they provide the Pravega Image information as part of the `PravegaCluster`